### PR TITLE
fix: improve summary edition aesthetic

### PR DESCRIPTION
**Pull request by Guilhem-lm**
Modifying the title input in order to not have it display as an input field.
* Before : 
image
image
image

* After : 
  * Empty
image
  * Hover
  Capture d’écran de 2024-06-28 09-04-39
  * focus
Capture d’écran de 2024-06-28 09-10-23
  * dark/light
image
image

----

| :rocket: | This description was created by [Ellipsis](https://www.ellipsis.dev) for commit 54294ba1b4e80d0aff39b8d13b7ea456571979f9  | 
|--------|--------|

### Summary:
Updated `ScriptBuilder.svelte` to change the appearance of the title input field using the new `app-title` class in `app.css`.

**Key points**:
* Updated `frontend/src/lib/components/ScriptBuilder.svelte` to change the appearance of the title input field.
* Added `app-title` class in `frontend/src/lib/assets/app.css` for styling the title input field.
* The `app-title` class removes the border and adds hover and focus effects.
* The title input field no longer displays as a traditional input field but as a styled text element.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

File: frontend/src/lib/assets/app.css [hunk 1]
```diff
@@ -104,3 +104,38 @@ svelte-virtual-list-contents > * + * {
 	@apply bg-surface;
 	@apply disabled:placeholder:text-gray-200 disabled:placeholder:dark:text-gray-500 disabled:text-gray-200 disabled:dark:text-gray-500 disabled:border-gray-200 disabled:dark:border-gray-600;
 }
+
+.app-title {
```
[the remark this answers is not in the document]
Comment (rubenfiszel) [reply frontend/src/lib/assets/app.css]: you should use the style inside the svelte element and not as a global app css. If wanting to make it global, you need to use tailwind.config.js

**Comment by rubenfiszel** [comment]
It's now hard to discover you can edit it by clicking on it. You at very least need a pen next to it or similar

**Comment by Guilhem-lm** [comment]
> It's now hard to discover you can edit it by clicking on it. You at very least need a pen next to it or similar

Yes I agree, i was thinking of changing the placeholder with "write a short summary here"

**Comment by rubenfiszel** [comment]
All the ideas are good but we use the clickable Pen icon next to a text to make it editable as a pattern in a few places, it's more consistent to keep it here and compatible with your other suggestions. Please apply it first so we can merge this

File: frontend/src/lib/assets/app.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Guilhem-lm) [reply frontend/src/lib/assets/app.css]: Done + pencil added 
image

**Pull request merged**